pubsub/topic.go

```diff
@@ -639,7 +639,7 @@ func (t *Topic) initBundler() {
 		// This is because there's no way to set "unlimited" for BufferedByteLimit,
 		// and simply setting it to MaxOutstandingBytes occasionally leads to issues where
 		// BufferedByteLimit is reached even though there are resources available.
-		t.PublishSettings.BufferedByteLimit = math.MaxInt64
+		t.PublishSettings.BufferedByteLimit = math.MaxInt
```

Comment:
Ah unfortunately this doesn't work since our min supported version is go1.15, `math.MaxInt` was added in 1.17. I believe the solution here is to detect whether the system uses 32 or 64 bits for ints using `strconv.IntSize` and then setting this value to either MaxInt32 or MaxInt64.

Reply:
Oh, that's a bummer. I went with option 2. Let me know if the comment is satisfactory.